minecraft_servers/lmcraft/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "server_name": "lmcraft",
+  "nice_name": "Lmcraft",
+  "direct_ip": "Lmcraft.de",
+  "server_wildcards": [
+    "%.Lmcraft.de"
+  ],
+  "social": {
+    "web": "https://Lmcraft.de",
+    "web_shop": "https://shop.Lmcraft.de",
```

Comment:
Not reachable

Reply:
Comming soon. Server is not puplic yet

Reply:
Then close this PR and reopen once your server is released. Only publicly available server can be added to the list.